Taskfile.Shared.yml

```diff
@@ -62,56 +62,31 @@ tasks:
     label: "{{.TASK}}-{{.NAME}}"
     dir: coprocessors/{{.NAME}}
     cmds:
-      - docker run -d --rm --name {{.NAME}}-co --env PORT="{{.COPROCESSOR_PORT}}" -m 1g --cpus=1 -p {{.COPROCESSOR_PORT}}:{{.COPROCESSOR_PORT}} {{.NAME}}-co
+      - docker run -d --rm --name coprocessor --net=task --env PORT="8000" -p 8000:8000 -m 1g --cpus=1 {{.NAME}}-co
     vars:
       NAME: "{{.NAME|default .CLI_ARGS}}"
   cleanup-coprocessor:
     label: "{{.TASK}}-{{.NAME}}"
     cmds:
-      - docker stop {{.NAME}}-co
+      - docker stop coprocessor
     vars:
       NAME: "{{.NAME|default .CLI_ARGS}}"
 
   start-subgraphs:
+    dir: subgraphs/fixed
     cmds:
-      - task: build-subgraph
-        vars:
-          NAME: a
-      - task: build-subgraph
-        vars:
-          NAME: b
-      - task: start-subgraph
-        vars:
-          NAME: a
-          PORT: 8081
-      - task: start-subgraph
-        vars:
-          NAME: b
-          PORT: 8080
+      - DOCKER_BUILDKIT=1 docker build . -t fixed
```

Comment:
For this, we can move this into another task (such as the above) to avoid rebuilding on each run. It's not a critical fix here since it can be a fast-follow, but helps when running locally.

Reply:
I have no idea how dependencies work in Taskfile, this is my first time 😅. So if the build task is a subtask it won't re-run when `start-subgraphs` is called from multiple tests? Or am I misunderstanding?

Reply:
For example:

```yml
  start-subgraphs:
    cmds:
      - task: build-fixed-subgraph
      - task: start-fixed-subgraph
  build-fixed-subgraph:
    dir: subgraphs/fixed
    cmds:
      - docker build . -t fixed
    sources:
      - "*.*"
  start-fixed-subgraph:
    dir: subgraphs/fixed
    cmds:
      - docker run --rm -d --name fixed -p 8082:8082 fixed
```

`build-fixed-subgraph` has a `sources` attribute; Task will generate a hash in a `.task` folder with a hash for the label and if it matches the current run, it'll skip the step. It's certainly a micro-optimization and BuildKit might actually fix this on Docker, but was just something we're doing generally to save a bit of time.